src/main/resources/application.properties

```diff
@@ -1,4 +1,4 @@
-#spring.datasource.url=jdbc:h2:mem:testdb
+spring.datasource.url=jdbc:h2:mem:testdb
```

Comment:
tento radek koliduje s radkem 3 (Oba definuji to same)

Reply:
Toto poprosím změnit pak co nejdřív.

Reply:
Done